src/multiformats/base.cljc

```diff
@@ -7,16 +7,15 @@
   (:refer-clojure :exclude [bases format])
   #?(:cljs
      (:require-macros
-       [multiformats.base :refer [defbase]]))
+      [multiformats.base :refer [defbase]]))
```

Comment:
The other namespaces are already styled correctly, so I'd prefer you didn't reformat them.